develop/devguide/Connector/SwarmingPrepare.md

```diff
@@ -36,6 +36,17 @@ This means that code in Automation scripts and connectors may need updates to ac
 > - Keep in mind that you will need to have access to the DataMiner 10.5.1 or 10.5.0 code base to apply these changes, as otherwise the updated/extended requests will not be available yet. However, in most cases, these updated/extended requests can be used to communicate with older server versions.
```

Comment:
(Actually, I don't believe we have any new overloads that result in a request that is incompatible with older server versions)